prisma/schema.prisma

```diff
@@ -53,7 +53,7 @@ model EventType {
   price                   Int                    @default(0)
   currency                String                 @default("usd")
   slotInterval            Int?
-  smartContractAddress               String?
+  smartContractAddress    String?
```

Comment:
is it still planned to move this into some sort of `metadata` (app related) table?

Reply:
In my understanding, yes. This will be moved to the app metadata table

Reply:
awesome :))